### ISSUE TITLE
[Next Release] Address the deprecation warnings

### ISSUE THREAD

**Issue by willingc** [jupyter/nbconvert#1382]
Prior to Python 3.9 release, we should review and address the current deprecation warnings.

**Comment by MSeal** [comment]
Yes, I think we can do a patch release to prep for 3.9 after this bug fix pass.
